docs/book/modules/admin/pages/resources.adoc

```diff
@@ -0,0 +1,47 @@
+= Resource requirements & configuration
+
+Proper resource requests and limits configuration in Kubernetes are important to ensure the application runs
+stable and performant. Therefore, there are some reasonable resource requests in the Helm charts, which get applied
+by default.
+
+TIP: To learn mor about Kubernetes resource management, read https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/[Resource Management for Pods and Containers].
+
+If your cluster does not meet these requirements, it may be that pods cannot be started and are stuck in a `Pending`
+state. If this is the case, you can either reduce the resource requirements, at the cost of stability, or provide more
+resources to the cluster, supporting the workload.
+
+== Default resource requests
+
+The default resource request is 1 CPU and 8 GiB of RAM, for both the importer and API server deployment. There are
+no resource limits by default.
```

Comment:
I suggest removing this H2 heading here, and just include these two sentences in your introduction section, after mentioning the Helm charts.  For example:

```
...
Therefore, there are some reasonable resource requests in the Helm charts, which get applied by default.
The default resource request is 1 CPU and 8 GiB of RAM, for both the importer and API server deployment.
There are no resource limits by default.
...
```